gnat2goto/ireps/follow.adb

```diff
@@ -12,4 +12,14 @@ package body Follow is
       end if;
       return Next;
    end Follow_Symbol_Type;
+
+   function Follow_Symbol_Type_Bounds (I : Irep; ST : Symbol_Table)
```

Comment:
??? What's this?

Reply:
There is now a separate function `Remove_Bounds` in the `ireps.ad*`.